sonic-ycabled/ycable/ycable_utilities/y_cable_helper.py

```diff
@@ -369,6 +372,40 @@ def retry_setup_grpc_channel_for_port(port, asic_index):
                 grpc_port_stubs[port] = stub
                 return True
 
+def apply_grpc_secrets_configuration(parsed_data):
```

Comment:
Better to put the file read logic into `apply_grpc_secrets_configuration`, so it could be used as:
```python
if os.path.isfile(SECRETS_PATH):
    apply_grpc_secrets_configuration(SECRETS_PATH)
```

Reply:
fixed